site/kubernetes/operator/install-operator.md

```diff
@@ -37,6 +37,9 @@ If you want to install a specific version of the Operator, you will have to obta
 [Operator Releases](https://github.com/rabbitmq/cluster-operator/releases). Please note that releases prior to 0.46.0
 do not have this manifest. We strongly recommend to install versions 0.46.0+
 
+If you can't pull from DockerHub, you will need to relocate the Operator image to a different location. The section [Relocate the Image](#relocate-image)
```

Comment:
There are reasons besides just inability to pull that could trigger image relocation. It probably makes sense to rephrase this sentence.

Reply:
Rephrased this sentence to make it more generic, without mentioning a specific reason to relocate.